tests/integration_tests/test_simple.py

```diff
@@ -0,0 +1,38 @@
+# This is an example test file. It won't get merged
+
+import pytest
+
+from tests.integration_tests.platforms import (
+    dynamic_client, OciClient, LxdContainerClient
+)
+
+
+CLOUD_CONFIG = """#cloud-config
+bootcmd:
+  - echo 'hello config!' > /tmp/cloud_config.txt"""
+
+
+class TestSimple:
+    @pytest.mark.cloud_config(CLOUD_CONFIG)
+    def test_cloud_config(self, client):
```

Comment:
If we have multiple tests that we want to run against a single instance, will this current design allow for that?

Reply:
Yeah, it's requires some small shenanigans in our conftest. I'll add them.